front/app/components/FormBuilder/components/FormFields/FormField/index.tsx

```diff
@@ -394,7 +394,7 @@ export const FormField = ({
             </Box>
           )}
         </FlexibleRow>
-        {showLogicOnRow && (
+        {showLogicOnRow && builderConfig.isLogicEnabled && (
           <Logic
```

Comment:
Fixes an existing issue where this shouldn't be shown if logic isn't enabled for the form.